domain/application/service/application.go

```diff
@@ -80,11 +80,11 @@ type ApplicationState interface {
 	// doesn't exist.
 	GetStoragePoolByName(ctx context.Context, name string) (domainstorage.StoragePoolDetails, error)
 
-	// UpsertCloudService updates the cloud service for the specified
-	// application, returning an error satisfying
-	// [applicationerrors.ApplicationNotFoundError] if the application doesn't
-	// exist.
-	UpsertCloudService(ctx context.Context, appName, providerID string, sAddrs network.SpaceAddresses) error
+	// UpsertCloudService updates the cloud service for the specified application.
+	//
+	// Returns an error satisfying [applicationerrors.ApplicationNotFoundError] if
+	// the application doesn't exist.
+	UpsertCloudService(ctx context.Context, appName, providerID string, sAddrs network.ProviderAddresses) error
```

Comment:
While you're here, could you update the error doc strings to follow the format in the styleguide?

https://github.com/juju/juju/blob/main/STYLE.md#method-documentation-errors